### PR TITLE
Fix some issues in `UpgradeToElectra`

### PR DESCRIPTION
**Pull request by jtraglia**
**What type of PR is this?**

Bug fix

**What does this PR do? Why is it needed?**

Primarily, this PR fixes some issues in `UpgradeToElectra`.

* ~~The logic for calculating `earliestExitEpoch` wasn't quite right.~~
  * ~~It was finding the max exit epoch, not the minimum.~~
  * ~~I believe my changes should work as intended, though it still deviates from the spec.~~
* `ExitBalanceToConsume` and `ConsolidationBalanceToConsume` should be set to 0.
  * These were being set to the total active balance.
  * The copy-paste spec for this says they should be zero, so not sure.
    * Speaking of this, the godoc function is out of date, but this PR doesn't fix that.
    * I've chatted with Kasey some about a long-term fix for this type of issue.
* Fix tests & rename some variables.
  * I don't think names like `lwvi` are very readable.

Also, some unrelated fixes:

* Delete unused `ProcessSyncAggregate` alias.
* In `PendingDeposit`, `Amount` was specified in the wrong order.

**Acknowledgements**

- [x] I have read [CONTRIBUTING.md](https://github.com/prysmaticlabs/prysm/blob/develop/CONTRIBUTING.md).
- [x] I have made an appropriate entry to [CHANGELOG.md](https://github.com/prysmaticlabs/prysm/blob/develop/CHANGELOG.md).
- [x] I have added a description to this PR with sufficient context for reviewers to understand this PR.

### CODE REVIEW

**Review by terencechain — COMMENTED** [review]
A bunch of spec tests are failing after this change. 
- TestMainnet_Electra_Transition
- TestMainnet_UpgradeToElectra

Are we sure it's the min? the spec looks max to me:
```python
  exit_epochs = [v.exit_epoch for v in pre.validators if v.exit_epoch != FAR_FUTURE_EPOCH]
    if not exit_epochs:
        exit_epochs = [get_current_epoch(pre)]
    earliest_exit_epoch = max(exit_epochs) + 1
```

**Comment by jtraglia** [comment]
> Are we sure it's the min? the spec looks max to me

Ah yes, you're right. I misread that. My fault. Will revert that change now.

File: beacon-chain/core/electra/upgrade.go [hunk 1]
```diff
@@ -268,9 +261,9 @@ func UpgradeToElectra(beaconState state.BeaconState) (state.BeaconState, error)
 
 		DepositRequestsStartIndex:     params.BeaconConfig().UnsetDepositRequestsStartIndex,
 		DepositBalanceToConsume:       0,
-		ExitBalanceToConsume:          helpers.ActivationExitChurnLimit(primitives.Gwei(tab)),
+		ExitBalanceToConsume:          0,
```
Comment (terencechain) [inline beacon-chain/core/electra/upgrade.go:264]: I dont think this is right either. We should assign them to churn limit based on total active balances
```
    post.exit_balance_to_consume = get_activation_exit_churn_limit(post)
    post.consolidation_balance_to_consume = get_consolidation_churn_limit(post)
```

Comment (jtraglia) [reply beacon-chain/core/electra/upgrade.go:264]: Ugh. Once again you're right. I'm going to close this.

**Pull request closed without merging**